Cargo.toml

```
@@ -4,7 +4,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
-bevy = "0.14.0"
+bevy = { git = "https://github.com/bevyengine/bevy/tree/main", rev = "9386bd0114c44c9f00a2e9c41db1225aaa78d159"}
```

```suggestion
bevy = { git = "https://github.com/bevyengine/bevy.git", rev = "9386bd0114c44c9f00a2e9c41db1225aaa78d159"}
```

This needs to be the path to the Git repository, not a specified branch, since `rev` already does so.

This fixes #31.

---

Ahhh I must've miss read the cargo docs when looking at how to specify the commit.